rust/src/traits/samplers/bernoulli/pseudocode/sample_bernoulli_float.py

```diff
@@ -0,0 +1,43 @@
+# type: ignore
+# returns a single bit with some probability of success
+def sample_bernoulli_float(prob: T, constant_time: bool) -> bool:
+    if prob == 1:  # |\label{line:1check}|
+        return True
+
+    # prepare for sampling first heads index by coin flipping
+    max_coin_flips = usize.exact_int_cast(T.EXPONENT_BIAS) + usize.exact_int_cast(
+        T.MANTISSA_BITS
+    )  # |\label{line:maxcoinflips}|
```

Comment:
The contents inside the pipe are escaped and parsed as latex. I put it behind a comment so as not to interfere with the python.

Reply:
Ok -- I see how the cross referencing works in the rendered PDF. I still wonder if these labels could be put at the end of the comment on the preceding line. 
- For a longer block of code, friendlier to link to the top rather than the bottom.
- As rendered, the `#` still comes through, and it looks a little weird in the PDF.